cnf-tests/testsuites/e2esuite/multinetworkpolicy/multinetworkpolicy_sriov.go

```diff
@@ -0,0 +1,819 @@
+		})
+
+		It("Allow access only to a specific port/protocol SCTP [sctp]", func() {
+
+			if !sctpEnabled {
+				Skip("SCTP not enabled on test nodes")
+			}
+
+			MakeMultiNetworkPolicy(TEST_NETWORK,
+				WithPodSelector(metav1.LabelSelector{
+					MatchLabels: map[string]string{
+						"pod": "a",
+					},
+				}),
+				WithIngressRule(multinetpolicyv1.MultiNetworkPolicyIngressRule{
+					Ports: []multinetpolicyv1.MultiNetworkPolicyPort{{
+						Port:     &port6666,
+						Protocol: &protoSCTP,
+					}},
+					From: []multinetpolicyv1.MultiNetworkPolicyPeer{{
+						PodSelector: &metav1.LabelSelector{
+							MatchLabels: map[string]string{
+								"pod": "b",
+							},
+						},
+					}},
+				}),
+				CreateInNamespace(nsX),
+			)
+
+			// Allowed
+			eventually30s(nsX_podB).Should(Reach(nsX_podA, OnPort(port6666), ViaSCTP))
+
+			// Not allowed
+			eventually30s(nsX_podB).ShouldNot(Reach(nsX_podA, OnPort(port6666), ViaUDP))
+			eventually30s(nsX_podB).ShouldNot(Reach(nsX_podA, OnPort(port6666), ViaTCP))
+			eventually30s(nsX_podB).ShouldNot(Reach(nsX_podA, OnPort(port5555), ViaTCP))
+			eventually30s(nsX_podB).ShouldNot(Reach(nsX_podA, OnPort(port5555), ViaUDP))
+			eventually30s(nsX_podB).ShouldNot(Reach(nsX_podA, OnPort(port5555), ViaSCTP))
+		})
+
+		It("Allow access only to a specific port/protocol TCP+UDP", func() {
+
+			MakeMultiNetworkPolicy(TEST_NETWORK,
+				WithPodSelector(metav1.LabelSelector{
+					MatchLabels: map[string]string{
+						"pod": "a",
+					},
+				}),
+				WithIngressRule(multinetpolicyv1.MultiNetworkPolicyIngressRule{
+					Ports: []multinetpolicyv1.MultiNetworkPolicyPort{{
+						Port:     &port5555, // Default protocol: TCP
+						Protocol: &protoTCP,
+					}, {
+						Port:     &port6666,
+						Protocol: &protoUDP,
+					}},
+					From: []multinetpolicyv1.MultiNetworkPolicyPeer{{
+						PodSelector: &metav1.LabelSelector{
+							MatchLabels: map[string]string{
+								"pod": "b",
+							},
+						},
+					}},
+				}),
+				CreateInNamespace(nsX),
+			)
+
+			// Allowed
+			eventually30s(nsX_podB).Should(Reach(nsX_podA, OnPort(port5555), ViaTCP))
+			eventually30s(nsX_podB).Should(Reach(nsX_podA, OnPort(port6666), ViaUDP))
+
+			// Not allowed
+			eventually30s(nsX_podB).ShouldNot(Reach(nsX_podA, OnPort(port6666), ViaTCP))
+			eventually30s(nsX_podB).ShouldNot(Reach(nsX_podA, OnPort(port5555), ViaUDP))
+		})
+
+		It("Allow access only to a specific UDP port from any pod", func() {
+
+			MakeMultiNetworkPolicy(TEST_NETWORK,
+				WithPodSelector(metav1.LabelSelector{
+					MatchLabels: map[string]string{
+						"pod": "a",
+					},
+				}),
+				WithIngressRule(multinetpolicyv1.MultiNetworkPolicyIngressRule{
+					Ports: []multinetpolicyv1.MultiNetworkPolicyPort{{
+						Port:     &port6666,
+						Protocol: &protoUDP,
+					}},
+				}),
+				CreateInNamespace(nsX),
+			)
+
+			// Allowed
+			eventually30s(nsX_podB).Should(Reach(nsX_podA, OnPort(port6666), ViaUDP))
+
```

Comment:
I think we should add also:
`eventually30s(nsX_podC).ShouldNot(Reach(nsX_podA, OnPort(port6666), ViaUDP))
`

Reply:
Cool! Added

Reply:
hmm.. Have you added?

Reply:
Sorry, I added in [`Allow access only to a specific port/protocol UDP`](https://github.com/openshift-kni/cnf-features-deploy/pull/1113/files#diff-b96c3f1d12439337ea75489fadce58f869013344d564ecbb6731fc0ba93a5f67R597), as it allows connection to `udp:6666` from `podB`

In the case `"Allow access only to a specific UDP port from any pod"`, podC can read podA on `udp:6666`